packages/cyberstorm/src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -3,41 +3,133 @@ import React, { PropsWithChildren } from "react";
 import { faHouse } from "@fortawesome/pro-regular-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import styles from "./BreadCrumbs.module.css";
-import { IndexLink } from "../Links/Links";
+import {
+  CommunitiesLink,
+  CommunityLink,
+  IndexLink,
+  PackageLink,
+  TeamsLink,
+} from "../Links/Links";
+
+type PageTitleCrumbProps = PropsWithChildren<{
+  pageTitle: string;
+}>;
 
 type BreadCrumbsProps = PropsWithChildren<{
-  excludeHome?: boolean;
+  pageTitle: string;
 }>;
 
-export function BreadCrumbs(props: BreadCrumbsProps) {
```

Comment:
I don't like dropping the generic `BreadCrumbs` component since we really don't want to have to define a separate component whenever we add new views. Why was this change done? What problem is it solving/bypassing? (Side note: the commit messages are quite unhelpful.)